src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -361,6 +361,55 @@ def load_arguments(self, _):
         c.argument('microsoft_account_client_secret', arg_group='Microsoft', help='AAD V2 Application client secret')
         c.argument('microsoft_account_oauth_scopes', nargs='+', help="One or more Microsoft authentification scopes (space-delimited).", arg_group='Microsoft')
 
+    with self.argument_context('webapp hybrid-connection list') as c:
+        c.argument('name', arg_type=webapp_name_arg_type, id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('slot', help="the name of the slot. Default to the productions slot if not specified")
```

Comment:
you can defined under `webapp hybrid-connection`, which will be picked up by any commands under